### ISSUE TITLE
PrivacyList Usage Help

### ISSUE THREAD

**Issue by UtkuDalmaz** [QuickBlox/quickblox-ios-sdk#887]
I am totally confused about how to use privacy list

My code is like

```
func chatDidReceive(_ privacyList: QBPrivacyList) {

        self.blockbtn.setTitle(NSLocalizedString("Block", comment: ""), for: UIControlState())

        privacyList.privacyItems.forEach { (item) in
            let privacyItem: QBPrivacyItem = item 
            let blockedid = privacyItem.userID
            let status = privacyItem.privacyType
            if (self.userQBID == UInt(blockedid) && Int(status.rawValue) == 1) {
                self.isblocked = true
                self.blockbtn.setTitle(NSLocalizedString("Unblock", comment: ""), for: UIControlState())
            }
        }

    }

    func chatDidReceivePrivacyListNames(_ listNames: [String]) {
        if (listNames.count == 0) {
           // let privacy = QBPrivacyItem(rawValue: 1)
            
            let item = QBPrivacyItem(privacyType: QBPrivacyType(rawValue: 1)!, userID: self.userQBID, allow: false)
            let list  = QBPrivacyList(name: "public", items: [item!])
            QBChat.instance.setPrivacyList(list)
            QBChat.instance.setDefaultPrivacyListWithName("public")
            self.isblocked = false
            self.blockbtn.setTitle(NSLocalizedString("Block", comment: ""), for: UIControlState())
        }
    } 
```
Can you please help using that Blocking system in a proper way?

**Comment by Pro100Andrey** [comment]
Hi @UtkuDalmaz, Could you please check and fill the template:

**New Issue Checklist**

 - [ ] I have read the [Documentation](https://quickblox.com/developers/IOS)
 - [ ] I have searched for a similar issue in the [project](https://github.com/QuickBlox/quickblox-ios-sdk/issues) and found none


**Environment details**

Info                           | Value                                       |
-------------------------------|---------------------------------------------|
iOS Version                    | e.g. 8.0                                    |                                          
Quickblox iOS SDK version      | e.g. 2.7.6                                  |
QuickbloxWebRTC SDK version    | e.g. 2.2.1                                  |                               
Xcode Version                  | e.g. Xcode 8.1                              |
Repro with our demo sample     | e.g. does it happen with our demo sample?   |

**Did this work before?**


**Expected behavior**


**Actual behavior**


**Logs**

*Please use the following template for logs:*

```
<details>
<summary>Title for logs </summary>
// Your logs here
</details>
```

To enable full logs use the following code:
```
[QBSettings setLogLevel:QBLogLevelDebug];
[QBSettings enableXMPPLogging];
```

**Steps to reproduce the behavior**


**Any others comments**

Thanks

**Comment by Pro100Andrey** [comment]
Hi @UtkuDalmaz, any updates ?

**Comment by VitaliyGuru** [comment]
This issue is being closed due to no response from our previous requests for information or action. We are assuming that this issue no longer exists. If this is still an issue, please open a new one and fill the template.
